components/grid/editing/validation.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Validation
+page_title: Grid - Validation
+description: Built-in validation in the Grid and how to customize the validation behavior.
+slug: grid-editing-validation
+tags: telerik,blazor,grid,validation,editing
+published: True
+position: 40
+---
+
+# Grid Validation
+
+The Telerik UI for Blazor Grid supports built-in validation That is enabled by default. The Grid passes down an `EditContext` as a cascading value to the editable cells in the component. If any validation messages are present the Grid will render them as [Validation Tooltips]({%slug validation-tools-tooltip%}) on hover of the specific editor. 
+
+#### In this article:
+
+* [Disable the validation](#disable-the-validation)
+
```

Comment:
We address a possible docs styling issue by using a separate list for each item. It's better to fix the styling, instead of using incorrect markup.
```suggestion
```